core/egov-workflow/src/main/java/org/egov/workflow/web/controller/RouterController.java

```diff
@@ -116,7 +116,7 @@
 
 	}
 	
-	@PostMapping(value = "/_update")
+	@PostMapping(value = "/{id}/_update")
```

Comment:
@gysenthil  there is {id} as path parameter for this controller method where is this id used?

If id is part of url then how do we handle role based action. We cannot insert actions for each id.